zap/src/main/java/org/zaproxy/zap/extension/alert/DialogModifyAlertTag.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2021 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.alert;
+
+import java.awt.Dialog;
+import org.parosproxy.paros.Constant;
+
+public class DialogModifyAlertTag extends DialogAddAlertTag {
```

Comment:
Should override the validation to accept the same key, otherwise you need to change the key always even if you just want to tweak the value.